### PR TITLE
use git url + check the version we print

### PR DESCRIPTION
**Pull request by mikemhenry**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by richardjgowers** [comment]
@mikemhenry is it possible to add a test that version == git_rev?

Will this get automatically triggered by new tags pushed to repos?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -47,6 +48,7 @@ test:
   commands:
     - pip check
     - openfe --help
+    - openfe --version
```
Comment (richardjgowers) [inline recipe/meta.yaml:51]: this checks the version via the cli, but I think they're defined in one place

Comment (mikemhenry) [reply recipe/meta.yaml:51]: I am not sure what you mean, this is how we get the version: https://github.com/OpenFreeEnergy/openfe/blob/main/openfecli/__init__.py#L8

Which is the same code path here: https://github.com/OpenFreeEnergy/openfe/blob/main/openfe/__init__.py#L26

**Comment by mikemhenry** [comment]
> @mikemhenry is it possible to add a test that version == git_rev?
> 
> Will this get automatically triggered by new tags pushed to repos?

I am not sure if the bot will auto make a PR when there is a new release with this method.

Another way to do this is to use pypi as the source for this because the version info is written at build time, so it will be there if we use a sdist to build the conda package.

**Pull request merged**